lib/dialogs/settings/panels/account.tsx

```diff
@@ -67,9 +80,17 @@ const AccountPanel = props => {
 
 AccountPanel.propTypes = {
   accountName: PropTypes.string.isRequired,
-  analyticsEnabled: PropTypes.bool.isRequired,
   requestSignOut: PropTypes.func.isRequired,
-  toggleShareAnalyticsPreference: PropTypes.func.isRequired,
 };
 
-export default AccountPanel;
+const mapStateToProps: S.MapState<StateProps> = ({
+  settings: { analyticsEnabled },
+}) => ({
+  analyticsEnabled,
+});
+
+const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
+  toggleAnalyticsEnabled: () => actions.settings.toggleAnalyticsEnabled(),
+};
```

Comment:
small thing, but I think it's worth keeping `mapDispatchToProps` in the simpler object form until we need to use the function form so that we can avoid some of the scenarios where it's so complex that it's confusing

```ts
const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
	toggleAnalyticsEnabled: actions.settings.toggleAnalyticsEnabled,
}
```

Reply:
For sure.